great_expectations/checkpoint/checkpoint.py

```diff
@@ -286,7 +290,10 @@ def is_fresh(self) -> CheckpointFreshnessDiagnostics:
 
         try:
             checkpoint = store.get(key=key)
-        except StoreBackendError:
+        except (
+            StoreBackendError,  # Generic error from stores
+            InvalidKeyError,  # Ephemeral context error
```

Comment:
We should follow up by patching the in-memory stores to return a consistent type